static/css/dashboard.css

```diff
@@ -22,9 +12,18 @@ body, .containers-dashboard {
   --dashboard-chart-title-FontWeight: var(--pf-global--FontWeight--normal);
 }
 
-.pf-c-tooltip {
-  /* make the tooltips look more like PF3 font and size */
-  --pf-c-tooltip__content--FontSize: 14px;
+body {
+  background-color: #F5F5F5;
+  font-size: 12px;
+  line-height: 1.66666667;
+  color: #363636;
+}
+
+#app {
+  min-height: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
 }
 
 #global-dashboard {
```

Comment:
Adding `width: 100%` here solved problems with resizing for me. With this the charts and heat maps fit the space. The only thing that is triggering overflow are the top cards (data centers, hosts, etc) but this is due to explicit `flex-wrap: nowrap` and seems to be this way before (not a regression).

Reply:
For getting narrow or wide?  I was playing with the responsive view and things look decent....

![oVirt Open Virtualization Manager - Google Chrome_068](https://user-images.githubusercontent.com/3985964/178746654-af59afd0-4b92-4cf6-8f37-13735b1d75a3.png)

![oVirt Open Virtualization Manager - Google Chrome_069](https://user-images.githubusercontent.com/3985964/178746679-d393977c-a153-4c41-9602-59254272bc50.png)

and wide looks ok as well...
![oVirt Open Virtualization Manager - Google Chrome_070](https://user-images.githubusercontent.com/3985964/178746908-a41ac128-ed6a-4f67-b975-cebafc131c66.png)

Reply:
https://user-images.githubusercontent.com/64194103/178750018-01bb739c-d01b-4c80-8759-0794c9c7f2c1.mp4

Reply:
On my env it looked differently - it's Chromium, dev mode with fake data.

Reply:
Ok so at least on Chromium, setting a `max-width: 100%` on the `#global-dishboard` very much helps.  It doesn't hurt the other browsers either.

Going down to a browser width of 970px with the left nav collapsed to icons still does not require horizontal scrolling. That covers most devices in landscape mode down to phones.

Changing the inventory status card to layout different under that size was always a point of contention.  There may be some reasonable ways to handle small widths.  I'll experiment a little bit.

Reply:
Looking good now:

[screencast-engine-dev.ovirt_8443-2022.07.14-21_49_34.webm](https://user-images.githubusercontent.com/3985964/179130757-97213355-7103-44fd-bce0-ea4a4c551025.webm)